packages/atlas/src/views/studio/VideoWorkspace/NftForm/AcceptTerms/AcceptTerms.tsx

```diff
@@ -0,0 +1,201 @@
+import { differenceInMilliseconds, format } from 'date-fns'
+import React from 'react'
+
+import { Banner } from '@/components/Banner'
+import { Information } from '@/components/Information'
+import { Text } from '@/components/Text'
+import { Checkbox } from '@/components/_inputs/Checkbox'
+import { useBlockTimeEstimation } from '@/hooks/useBlockTimeEstimation'
+
+import { Description, Divider, Header, Row, StyledSvgActionArrowRight, TermsBox, Title } from './AcceptTerms.styles'
+
+import { StyledSvgWarning, YellowText } from '../../VideoWorkspace.style'
+import { useNftForm } from '../NftForm.hooks'
+import { AuctionDurationTooltipFooter } from '../NftForm.styles'
+import { RoyaltiesTooltipFooter } from '../RoyaltiesTooltipFooter'
+import { Listing, NftFormData } from '../types'
+
+type AcceptTermsProps = {
+  selectedType: Listing
+  formData: NftFormData
+  termsAccepted: boolean
+  toggleTermsAccept: () => void
+}
+
+const DATE_FORMAT = 'dd MMM yyyy, HH:mm OOOO'
+
+export const AcceptTerms: React.FC<AcceptTermsProps> = ({
+  selectedType,
+  formData,
+  termsAccepted,
+  toggleTermsAccept,
+}) => {
+  const { getTotalDaysAndHoursText } = useNftForm()
+  const { convertDurationToBlocks } = useBlockTimeEstimation()
+
+  const startDate = formData.startDate as Date
+  const endDate = formData.endDate as Date
+
+  const validDate = formData.startDate instanceof Date && formData.endDate instanceof Date
```

Comment:
Safer to use `isValid` from `date-fns`, it checks other stuff than just being a `Date` object